django/db/models/fields/__init__.py

```diff
@@ -237,14 +237,15 @@ def _check_field_name(self):
         else:
             return []
 
+    @classmethod
```

Comment:
If you're using `@classmethod` because `self` wasn't used, then you may as well use `@staticmethod` as `cls` won't be used either...

Reply:
Good point, Thanks.